### ISSUE TITLE
Indicate debugging support for .NET Framework

### ISSUE THREAD

**Issue by scottaddie** [Microsoft/vscode-docs#1339]
https://code.visualstudio.com/docs/languages/csharp

This C# doc contains the following note:
> **Note:** VS Code does not support debugging applications running on the Desktop .NET Framework.

The following merged PR makes that note a false statement:
https://github.com/omnisharp/omnisharp-vscode/pull/1390

@WardenGnaw Under what conditions is debugging in VS Code supported for .NET Framework? Does the app have to be running in a 64-bit process and must portable PDBs be used? Also, is this only supported for ASP.NET Core apps?

**Comment by WardenGnaw** [comment]
Thanks for pointing out that note! I didn't know that was there. 

There is currently limited support for full .NET framework debugging. However, it should work for any app with 64-bit process and portable PDBs.

I'll be working on a wiki page on the omnisharp-csharp repo and make a PR here to link to it.

**Comment by gregvanl** [comment]
Hi @WardenGnaw Is there a vscode-docs PR pending? If so, will it address this issue?

**Comment by gregvanl** [comment]
Merged #1372, closing

**Comment by dzmitry-lahoda** [comment]
First task works, second fails.

```
        {
            "name": ".NET Framework Attach (FooBar)",
            "type": "clr",
            "request": "attach",
            "processName": "FooBar.exe",
        },
        {
            "name": ".NET Framework Launch (FooBar)",
            "type": "clr",
            "request": "launch",
            "cwd": "${workspaceFolder}/",
            "program": "${workspaceFolder}/FooBar.exe",
            "args":"/config FooBar.Dev.config",         
        },
```
Error is next:
``
WARNING: Could not load symbols for 'FooBar.dll'. 'FooBar.pdb' is a Windows PDB. These are not supported by the cross-platform .NET Core debugger.
Loaded 'FooBar.dll'. Cannot find or open the PDB file.
```

Not sure how to ask to attach `clr` debugger, not `coreclr`. I have typed ` "type": "clr"` already.
